projects/canopy/src/lib/button/button.notes.ts

```diff
@@ -39,11 +21,7 @@ Content projection is used as Canopy Icons need loading via the \`iconRegistry\`
 </button>
 ~~~
 
-## Button with icon only
-
-Buttons can be setup as icon only buttons, this is done in a similar way to a button with icon and text.
-You will still need to include text content inside the button as this information is vital to screen readers.
-The \`iconButton\` property is used to visually hide that text.
+You can create an **Icon Only** Button by setting \`iconButton\` to \`true\`. You must still include text content, which will be visually hidden, but detectable by screen readers.
```

Comment:
I wouldn't capitalise the `Icon Only Button` 🤔

Reply:
Agreed, it looks better without that, removed ✅